client/app/models/disposition.js

```diff
@@ -121,50 +121,9 @@ export default class DispositionModel extends Model {
 
   @attr('string', { defaultValue: '' }) dcpBoroughpresidentrecommendation;
 
-  // sourced from statuscode
-  // Label: 'Draft', 'Value': 1
-  // Label: 'Saved', 'Value': 717170000
-  // Label: 'Submitted', 'Value': 2
-  // Label: 'Deactivated', 'Value': 717170001
-  // Label: 'Not Submitted', 'Value': 717170002
-  @attrComputed(
-    // recommendations
-    'dcpDateofvote',
-    'dcpBoroughpresidentrecommendation',
-    'dcpBoroughboardrecommendation',
-    'dcpCommunityboardrecommendation',
-    'dcpIspublichearingrequired',
-
-    // hearings
-    'dcpPublichearinglocation',
-    'dcpDateofpublichearing',
-  )
-  get statuscode() {
```

Comment:
I'm pretty concerned about removing this logic because of how it might affect the LUP side of things